app/services/search/postgres/comment.rb

```diff
@@ -55,6 +47,7 @@ def self.search_documents(
         per_page = [(per_page || DEFAULT_PER_PAGE).to_i, MAX_PER_PAGE].min
 
         relation = ::Comment
+          .includes(:user)
```

Comment:
IIRC the idea originally behind not including user was the size of the objects (many columns in this table). 

Just reminding myself of tradeoffs we're making, not questioning your choice :)